database/rrdset.c

```diff
@@ -659,22 +649,6 @@ RRDSET *rrdset_create_custom(
         }
 
         if (mark_rebuild) {
-#ifdef ENABLE_ACLK
-            if (netdata_cloud_setting) {
-                if (mark_rebuild & META_CHART_ACTIVATED) {
-                    aclk_add_collector(host, st->plugin_name, st->module_name);
-                }
-                else {
-                    if (mark_rebuild & (META_PLUGIN_UPDATED | META_MODULE_UPDATED)) {
-                        aclk_del_collector(
-                            host, mark_rebuild & META_PLUGIN_UPDATED ? old_plugin : st->plugin_name,
-                            mark_rebuild & META_MODULE_UPDATED ? old_module : st->module_name);
-                        aclk_add_collector(host, st->plugin_name, st->module_name);
-                    }
-                }
-                rrdset_flag_clear(st, RRDSET_FLAG_ACLK);
```

Comment:
We need to keep the `rrdset_flag_clear(st, RRDSET_FLAG_ACLK);` when the metadata changes.